assignment/api/src/resources/assignment.properties

```
@@ -254,7 +254,9 @@ gen.yoursubwill.indexed = Your submission will be sent to {0} to be reviewed for
 gen.thesubswill   = The submissions will be sent to {0} to be reviewed for plagiarism.
 gen.thesubswill.indexed = The submissions will be sent to {0} to be reviewed for plagiarism. They will be indexed to be compared against by other submissions.
 gen.subStudentPreview = This assignment has enabled student preview which allows students to submit as many papers as a draft as they wish. A draft paper report is only visible to the owner of the paper, however instructors will be able to see the number of draft submissions and the report scores of each draft. Also, if there is a due date, the most recent draft will automatically be submitted if the student has no other submissions. Students can submit a draft paper as their final submission at any time within the plagiarism report.
-gen.onlythefoll   = Only the following file types will be accepted: {0}
+gen.onlythefoll   = <li>Only the following file types will be accepted: {0}.</li>
+gen.onlythefoll.turnitin = <li>File extension should always be included. Text after first point will be considered as file extension.</li><li>Attachments should not exceed 100 MB in size (.txt attachments should not exceed 2 MB).</li><li>Attachments should occupy less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
```

Is this true that text after first dot will be the file extension?! They don't use the last dot? 

Also line 792 uses "dot" and line 258 uses "point"

---

Agreed. I would have preferred this stayed open a little longer so the language could be reviewed by more than one person.

---

No problem to make text updates, sorry for that guys, @josecebe will work on that. any preferred option or suggestion?

---

```suggestion
gen.onlythefoll.turnitin = <li>The uploaded file must have a file extension, which is the text following the dot.</li><li>Attachments must be less than 100 MB in size (.txt attachments should be less than 2 MB), have a minimum of 20 words, and contain less than 800 pages.</li>
```

Where is the vector image recommendation coming from? JPEG doesn't seem like a great type to be recommending for a comparison because of its compression, which can cause blurriness.

---

I believe the text about the vector images comes from the UPV experience, @josecebe and @vacercon anything to say about this?

---

These messages display regardless of which content review provider you use, so the messages must be agnostic of which provider is in use. If you need specific messaging for a specific provider, then it should be implemented in the provider code.

---

As I see it states turnitin?

---

My mistake.